integration_test/third_party_apps_data/applications/hbase/centos_rhel/install

```diff
@@ -0,0 +1,36 @@
+set -e
+
+sudo yum -y install \
+    java-1.8.0-openjdk java-1.8.0-openjdk-devel wget
+
+sudo mkdir /opt/hbase
+wget http://apache.mirror.gtcomm.net/hbase/stable/hbase-2.4.9-bin.tar.gz
+sudo tar -xvf hbase-2.4.9-bin.tar.gz -C /opt/hbase --strip 1
+
+JAVA_HOME="$(dirname "$(dirname "$(readlink "$(readlink "$(which javac)")")")")"
+
+cat <<EOF | sudo tee /etc/systemd/system/hbase.service
```

Comment:
I looked through the docs and the release tarball and I can't for the life of me find a single set of instructions for how you are supposed to start hbase on boot. Where did you find this systemd unit? Can you add a link in a comment?

Reply:
Apache doesn't provide a service file, we based ours off of [this](https://github.com/esamson/baseboxes/blob/master/sl-tools/etc/systemd/system/hbase.service).

Reply:
Interesting. I will be very happy when we move on to non-Java software that has more standard installation/configuration mechanisms!